piet-cairo/src/lib.rs

```diff
@@ -269,6 +270,11 @@ impl<'a> RenderContext for CairoRenderContext<'a> {
                             data[dst_off + x * 4 + 3] = a;
                         }
                     }
+                    ImageFormat::Grayscale => {
```

Comment:
It seems to me it should be possible to avoid this. I see a TODO in #350 regarding this. Given that it's reported to work, I'm inclined to accept this now, and consider a more efficient render path to be potential followup work.

Reply:
The more I think about this the more I think it's just incorrect, and pushes too much expectation down to the consumer; the behaviour coming out of piet should be the same for all backends, which means we should just use rgb24. :shrug:

Reply:
Wait so instead you are quadrupling the data? How is this more efficient? I am dealing with 500MB+ images, this looks very bad to me.